cmd/storage.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/storage/cp"
+	"github.com/supabase/cli/internal/storage/ls"
+	"github.com/supabase/cli/internal/storage/mv"
+	"github.com/supabase/cli/internal/storage/rm"
+)
+
+var (
+	storageCmd = &cobra.Command{
+		GroupID: groupManagementAPI,
+		Use:     "storage",
+		Short:   "Manage Supabase Storage objects",
+	}
+
+	recursive bool
+
+	lsCmd = &cobra.Command{
+		Use:     "ls [prefix]",
+		Example: "ls ss:///bucket/docs",
+		Short:   "List objects by path prefix",
+		Args:    cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			objectPath := ls.STORAGE_SCHEME + ":///"
+			if len(args) > 0 {
+				objectPath = args[0]
```

Comment:
Not sure what's the point of having a default `objectPath` here, as it will fail anyway due to `ParseStorageURL` check.

Reply:
Unless this is actual "root" bucket and `ParseStorageURL` will treat it as literal `/`

Reply:
This is meant to be a convenience for listing all buckets or objects, for eg.
```
$ supabase storage ls
test/
nothing/
private/
```